omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java

```diff
@@ -155,12 +155,12 @@ private String getRedirectUrlFromRequest(PageRequest pageRequest){
 
 	private String getRedirectUrl(PageRequest pageRequest){
 		String redirectUrl = getRedirectUrlFromRequest(pageRequest);
-		if(StringUtils.isBlank(redirectUrl)) {
-			redirectUrl = getRedirectUrlFromReferer(pageRequest);
-		}
-		if(StringUtils.isBlank(redirectUrl)) {
-			redirectUrl = getStringSessionAttribute(SESSION_ATTRIBUTE_REDIRECT_URL, pageRequest.getRequest());
-		}
```

Comment:
I saw a lot of code formatting issues while working with Open Source projects. We can have a predefined code formatting standard or IDE preference to avoid such issues.

http://programmers.stackexchange.com/questions/189274/is-imposing-the-same-code-format-for-all-developers-a-good-idea